src/jdk.jfr/share/classes/jdk/jfr/internal/tracing/PlatformTracer.java

```diff
@@ -46,6 +46,8 @@
  * the jdk.jfr.tracing package fewer internals are exposed to the application.
  */
 public final class PlatformTracer {
+    private static volatile boolean INITIALIZED;
```

Comment:
Can we make this a static final field in a holder class, so we call `PlatformTracer::initialize` from that holder class's `<clinit>`, and after initialization, the check to `INITIALIZED` field can be constant-folded instead of being a volatile read?